docs/knowledge/searching-on-github.md

```
@@ -0,0 +1,147 @@
+| Qualifier                                         | Example                                                                                                                                                                                                            |
+| ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| <code>involves:<em>USERNAME</em></code>           | **[involves:defunkt involves:jlord](https://github.com/search?q=involves%3Adefunkt+involves%3Ajlord\&type=Issues)** matches issues either @defunkt or @jlord are involved in.                                      |
+| `in:body` <code>involves:<em>USERNAME</em></code> | [**NOT bootstrap in:body involves:mdo**](https://github.com/search?q=NOT+bootstrap+in%3Abody+involves%3Amdo\&type=Issues) matches issues @mdo is involved in that do not contain the word "bootstrap" in the body. |
+
+## Search for my issues and pull requests
+
+You can search for issues and pull requests you have created or have interacted with by following the desired qualifier with `@me`. Any qualifier that works with a username allows you to limit your search to issues and pull requests you created, are assigned, mentioned on, or are requested as a reviewer of.
+
+| Qualifier                                 | Example                                                                                                                                          |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| <code>author:<em>@me</em></code>          | **[author:@me](https://github.com/search?q=author%3A%40me)** matches issues and pull requests yo
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)